index.html

```diff
@@ -75,7 +77,7 @@ <h1>Chocolate Pizza</h1>
     <h2>Vanessa Stevenson</h2>
     <p>Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p>
     <button>SHARE RECEIPE</button>
-    <img src="img/small-logo.png" alt="Small gray logo" title="small gray logo" />
+    <img id="graylogo" src="img/small-logo.png" alt="Small gray logo" title="small gray logo" />
     <p>Delicious &copy 2013 ~ All Rights Reserved.</p>
     <p>Proudly published with Ghost.</p>
```

Comment:
It might be easier to organize your footer if the bio and the copyright are in separate boxes.
```html
<div>
     <h2>Vanessa Stevenson</h2>
     <p>Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p>
     <button>SHARE RECEIPE</button>
</div>
<div>
     <img id="graylogo" src="img/small-logo.png" alt="Small gray logo" title="small gray logo" />
     <p>Delicious &copy 2013 ~ All Rights Reserved.</p>
     <p>Proudly published with Ghost.</p>
</div>
```